site/src/components/WorkspaceScheduleButton/WorkspaceScheduleButton.stories.tsx

```diff
@@ -87,3 +87,9 @@ CannotEdit.args = {
   },
   canUpdateWorkspace: false,
 }
+
+export const SmallViewport = Template.bind({})
+SmallViewport.args = WorkspaceOffShort.args
+SmallViewport.parameters = {
+  chromatic: { viewports: [320] },
```

Comment:
This looks useful, how does it work?

Reply:
We basically are saying to chromatic to test this snapshot using the 320px viewport. Unfortunately, looks like a Chromatic thing not a Storybook native option.

Reply:
Looks like maybe Chromatic isn't registering this one?